tests/.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{"C:\\Maintraq\\app\\__init__.py":[1,3,4,5,6,7],"C:\\Maintraq\\app\\views2.py":[1,2,3,5,6,7,8,9,10,11,14,16,20,21,23,46,58,78,108,109,110,47,56,48,55,24,44,132,133,134,141,142,143,148,149,150,176,177,178,185,186,187,194,195,196,204,205,206,214,215,216,61,62,66,68,69,63,81,82,83,84,85,86,90,93,96,97,98,99,100,71,72,75,87,88,94,26,27,31,32,33,34,35,36,37,43,50,54,114,115,116,117,120,121,122,123,124,125,126,138,139,154,156,157,158,159,145,146],"C:\\Maintraq\\app\\models2.py":[1,2,3,4,5,6,8,10,11,19,31,52,53,54,62,12,13,14,15,23,25,36,38,39,40,41,42,44,45,46,47,55,56,57,58,64,65],"C:\\Maintraq\\app\\helpers.py":[1,2,4,5,6,7,8,9,12,14,16,22,32,41,47,57,62,69,74,81,87,92,97,104,42,43,44,17,18,20,58,59,60,61,63,64,65,67,70,71,72],"C:\\Maintraq\\app\\views.py":[1,2,3,4,5,6,7,8,9,11,13,15,16,19,40,63,86,87,20,38,109,110,118,119,144,145,152,153,165,166,43,44,45,47,49,51,52,53,54,55,56,57,58,66,67,68,70,72,73,74,75,76,77,79,80,22,23,27,28,29,30,31,37,90,91,92,94,98,99,100,101,102,103,169,170,171,172,173,174,175,176,50,177,82,83,84,71,69,34,35,156,157,158,159,160,161,33,24,25,26,59,60,61,46,48,95,96,93,113,114,115,116,123,124,126,127,128,129,130,131,133,134,135,136],"C:\\Maintraq\\app\\models.py":[1,3,5,9,18,19,20,23,6,7,21,13,14,15,16,25,27,28,29,30,31,33,34]}}
```

Comment:
remove this file and add it to a gitignore.

Reply:
thanks, implementing the changes right away